src/AppBundle/Entity/PersonType.php

```diff
@@ -36,14 +37,15 @@ class PersonType
      *
      * @ORM\Column(type="string")
      * @Assert\Type(type="string")
+     * @Iri("https://schema.org/name")
      * @Groups({"person_type_read", "person_type_write"})
      */
     private $name;
 
     /**
      * @var ArrayCollection
      *
-     * @ORM\OneToMany(targetEntity="Person", mappedBy="type")
+     * @ORM\OneToMany(targetEntity="Person", mappedBy="personType")
      * @Groups({"person_type_read", "person_type_write"})
      */
     private $persons;
```

Comment:
Don't duplicate the data. We can do it with sql request. I think it's better to remove this property